backend/Squirrel.ConsoleApp.BL/Services/ConnectionFileService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models.Models;
+
+namespace Squirrel.ConsoleApp.BL.Services;
+
+public class ConnectionFileService: IConnectionFileService
+{
+    public void CreateEmptyFile()
+    {
+        var filePath = GetFilePath();
+        if (!File.Exists(filePath))
+        {
+            File.WriteAllText(filePath, "{}");
+        }
+    }
+
+    public ConnectionString ReadFromFile()
+    {
+        var filePath = GetFilePath();
+        if (File.Exists(filePath))
+        {
+            var json = File.ReadAllText(filePath);
+            return JsonConvert.DeserializeObject<ConnectionString>(json) ?? throw new InvalidOperationException();
+        }
+
+        throw new FileNotFoundException(filePath);
+    }
+
+    public void SaveToFile(ConnectionString connectionString)
+    {
+        var filePath = GetFilePath();
+        string json = JsonConvert.SerializeObject(connectionString, Formatting.Indented);
+        File.WriteAllText(filePath, json);
+    }
+
+    private string GetFilePath()
+    {
+        string userFolder = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
+        return Path.Combine(userFolder, "squirrel-connection.json");
+    }
+}
```

Comment:
you use this method in every public method, so maybe it better to create a property `FilePath` with this method in getter